src/Collada.cpp

```diff
@@ -589,6 +589,8 @@ TiXmlElement* Collada::createNode(string name, TiXmlElement* parentNode, Vector3
 
 	if (instanceGeometryName.length() != 0)
 		makeInstanceGeometryOnNode(node, instanceGeometryName, materialName, haveTexture);
+  else
+  	node->SetAttribute("type", "NODE");
```

Comment:
not 100% sure about this, but this is what blender files had.

I don't intend to change this.